m4/lx_find_lustre.m4

```diff
@@ -7,18 +7,20 @@ AC_DEFUN([X_AC_LUSTRE], [
                 [], [enable_lustre="no"])
   AC_MSG_RESULT([$enable_lustre])
 
-  AC_MSG_CHECKING([for Lustre])
-  AS_IF([test -e "/usr/include/lustre/lustre_user.h" && test -e "/usr/include/lustre/lustreapi.h"],
-        [enable_lustre="yes"],
-        [enable_lustre="no"])
-  AC_MSG_RESULT([$enable_lustre])
+  if [[ "x$enable_lustre" == xyes ]]; then
+    AC_MSG_CHECKING([for Lustre])
+    AS_IF([test -e "/usr/include/lustre/lustre_user.h" && test -e "/usr/include/lustre/lustreapi.h"],
+          [lustre_available="yes"],
+          [enable_available="no"])
```

Comment:
Should this line be `[lustre_available="no"])`?

Reply:
@Natale, thanks, it's typo, let me fix it and push again